src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/Statics.cs

```diff
@@ -169,7 +169,7 @@ public static void EncodeTags(int tags, ref int pos, byte[]? metadata)
 
         public static void CheckName(string? name)
         {
-            if (name != null && 0 <= name.IndexOf('\0'))
+            if (name != null && name.Contains('\0'))
```

Comment:
This is causing me issues and I am not sure how to fix it so that it compiles.

Reply:
@danmosemsft this is what is causing the build failure. If we use IndexOf then the code is marked by one of the analyzers with a warning, causing a build failure. If we use contains, then the net48 build fails. Since net48 doesn't have an overloads for char.

Reply:
These tracing files are built into 2 projects, one is corelib and the other is a redist project for .NET Framework use. This second one is causing the problem here. 
https://dev.azure.com/dnceng/public/_build/results?buildId=779866&view=logs&j=ed31f2e8-9a21-528b-426c-e91b77680964&t=0cc22ee9-e6e6-58fc-cb15-c6bbf3030f86&l=930 
the fix is to check for ES_BUILD_STANDALONE and if it's set then use the .NET Framework version
eg 
https://github.com/danmosemsft/runtime/blob/9646df5d76213e673c2cb44aced17113a4961a01/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs#L3797